.ci/azure/linux.yml

```diff
@@ -99,3 +112,24 @@ jobs:
     env:
       LD_LIBRARY_PATH: $(BIN_DIR)/lib
     displayName: 'Java tests'
+
+  - script: |
+      sudo apt-get install -y python3-setuptools
+      python3 -m pip install \
+          numpy \
+          defusedxml \
+          networkx \
+          test-generator==0.1.1 \
+          torch \
+          torchvision \
+          opencv-python
+
+      git clone --depth 1 https://github.com/facebookresearch/detectron2
```

Comment:
it is preferable to set stable version. Tests should be reproducible

Reply:
I just did as we discussed here: https://github.com/openvinotoolkit/openvino_contrib/pull/10#discussion_r528174552

Reply:
Sorry, misunderstood you point. Done